demo.html

```diff
@@ -0,0 +1,442 @@
+                        <option value="Wallis and Futuna Islands">Wallis and Futuna Islands</option>
+                        <option value="Western Sahara">Western Sahara</option>
+                        <option value="Yemen">Yemen</option>
+                        <option value="Zambia">Zambia</option>
+                        <option value="Zimbabwe">Zimbabwe</option>
+                    </select>
+                </div>
+                <div class="form-group">
+                    <label for="form-role">Industry *</label>
+                    <select class="form-control" id="form-industry" name="industry">
+                        <option value="" selected="selected">Select</option>
+                        <option value="Higher Ed. - For Profit Ed.">Higher Ed. - For Profit Ed.</option>
+                        <option value="Higher Ed. - Not for Profit Ed.">Higher Ed. - Not for Profit Ed.</option>
+                        <option value="Higher Ed. - Other Vocation Education">Higher Ed. - Other Vocation Education</option>
+                        <option value="Further Education">Further Education</option>
+                        <option value="K12">K12</option>
+                        <option value="Corporate">Corporate</option>
+                        <option value="Non-Corporate Organization">Non-Corporate Organization</option>
+                        <option value="Government - DoD">Government - DoD</option>
+                        <option value="Government - Other">Government - Other</option>
+                    </select>
+                </div>
+                <div class="form-group">
+                    <label for="form-phone">Phone</label>
+                    <input type="text" class="form-control" id="form-phone" name="businessPhone" placeholder="Your phone number">
+                </div>
+
+                <button class="primary" type="submit">Request demo</button>
+                <p>By submitting this form you agree to receiving emails from Blackboard. You can unsubscribe at any time. Please visit our <a href="https://www.blackboard.com/footer/privacy-policy.aspx" title="Blackboard Privay Policy" >Privacy Policy</a> for more information.
+
+
+                <input value="AllyMicrosite" type="hidden" name="elqFormName" />
+                <input value="2376" type="hidden" name="elqSiteId" />
+                <input name="elqCampaignId" type="hidden" />
+                <input id="field16" type="hidden" name="utm_source" value="" />
+                <input id="field17" type="hidden" name="utm_medium" value="" />
+                <input id="field18" type="hidden" name="utm_term" value="" />
+                <input id="field19" type="hidden" name="utm_content" value="" />
+                <input id="field20" type="hidden" name="utm_campaign" value="" />
+                <input id="field11" type="hidden" name="numberOfUsers" value="0-100" />
```

Comment:
Remove "Fronteer" and "0-100" values:
`<input id="field11" type="hidden" name="numberOfUsers" value="n/a" />
<input id="field4" type="hidden" name="company" value="n/a"  />`